Userland/Libraries/LibGUI/Calendar.h

```diff
@@ -75,12 +72,15 @@ class Calendar final : public GUI::Frame {
     Function<void()> on_month_click;
 
 private:
+    Calendar(Core::DateTime date_time = Core::DateTime::now(), Mode mode = Month);
+    virtual ~Calendar() override;
+
     virtual void resize_event(GUI::ResizeEvent&) override;
     virtual void paint_event(GUI::PaintEvent&) override;
     virtual void mousemove_event(GUI::MouseEvent&) override;
     virtual void mousedown_event(MouseEvent&) override;
     virtual void mouseup_event(MouseEvent&) override;
-    virtual void doubleclick_event(MouseEvent&);
+    virtual void doubleclick_event(MouseEvent&) override;
```

Comment:
Did this not trigger a warning? We should check that it does.

Reply:
It did not. I only did these fixes as a drive-by, so I don't really want to write a new check for this.